styles/mixins/_base.scss

```diff
@@ -0,0 +1,16 @@
+/**
+ * Override CSS rules.
+ *
+ * This mixin serves two purposes:
+ * - Allows developers to be explicit that the contained CSS rule-set overrides
+ *   styling imposed by another pattern
+ * - Increases the specificity of the contained rule-set to aid in overriding
+ *
+ * Current implementation: Use the :not pseudo-class to increase specifity.
+ */
+@mixin override() {
+  // The ID here will increase the specificity of the whole selector
+  &:not(#increase-specificity) {
```

Comment:
Interesting.

Reply:
Yah. I want to be sure this isn't super confusing. So please speak up if it just seems crazy pants.